flask_monitoringdashboard/core/config/__init__.py

```diff
@@ -43,7 +43,11 @@ def __init__(self):
 
         # visualization
         self.colors = {}
-        self.timezone = pytz.timezone(str(get_localzone()))
+        try:
+            default_timezone = pytz.timezone(str(get_localzone()))
+        except pytz.UnknownTimeZoneError:
+            default_timezone = pytz.timezone('UTC')
```

Comment:
Is there a way we can log some sort of warning when this happens? I guess it makes sense to not magically handle misconfigured systems.

Reply:
Hi @nstapelbroek, 

Thanks for your PR. I've slidely modified your PR and changed the base into development branche.

Thanks for your contribution, it will be accepted soon!

Patrick